tensorflow/tools/ci_build/builds/check_system_libs.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Checks that the options mentioned in syslibs_configure.bzl are consistent with the valid options in workspace.bzl
+# Expects the tensorflow source folder as the first argument
+
+import sys
+import os
+from glob import glob
+
+tf_source_path = sys.argv[1]
+
+if not os.path.isdir(tf_source_path):
```

Comment:
Maybe lets move this under the path joins then the check can be `isdir(tf_source_path) and isfile(syslibs_configure_path)` to make sure the arg is the right dir not just any dir